### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by hadim** [nglviewer/nglview#1055]
The latest `main` is compatible with ipywidgets=8. Would that be possible to release a new version @hainm ?

**Comment by hainm** [comment]
Dear @hadim,

I hesitate to do that since there is some issues with building the extension for jupyterlab.
I hope someone can help with this.

```
- cd js && npm install # causes error
- if error is fixed, make sure nglview can work (again) with latest lab
```

**Comment by hainm** [comment]
Here is the release procedure (I hope it's not outdated yet): 
https://github.com/nglviewer/nglview/blob/master/devtools/README.release

**Comment by hainm** [comment]
um, I give up jupyterlab for now and will focus on the release just for using nglview and ipywidgets 8 with notebook for now. 

Ran tests with ipywidgets 8 and got some failures. Working on the fix.

**Comment by hainm** [comment]
pip version is ready: <img width="456" alt="image" src="https://user-images.githubusercontent.com/4451957/235283012-fb9c14b1-8516-4bbc-8424-a217fa1204c2.png">

**Comment by hadim** [comment]
Thanks. I confirm it works for notebook but not JupyterLab. I, personally, use only Jupyter lab, and it's likely the case for many other users.

I am not very familiar with the JLab build system. I tried to build it locally but it fails during ` jupyter labextension link . --debug`.

**Comment by hainm** [comment]
> I, personally, use only Jupyter lab, and it's likely the case for many other users.

yeah, I am aware of that. I find it hard catching up with jupyter lab.
